l10n/en/brands.ftl

```diff
@@ -129,6 +129,7 @@
 -brand-name-windows = Windows
 -brand-name-xp = XP
 -brand-name-vista = Vista
+-brand-name-chromeos = ChromeOS
```

Comment:
Should it be `Chrome OS`? https://www.google.com/intl/en_uk/chromebook/chrome-os/